landing/login/login.factory.js

```diff
@@ -0,0 +1,21 @@
+(() => {
+
+    'use strict';
+
+    angular
+        .module('app')
+        .factory('Login', Login);
+
+    Login.$inject = ['$resource', '$log', 'Endpoints'];
+    function Login($resource, Endpoints) {
+        return $resource(Endpoints.API.AUTH, null, {
+            'send': {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                isArray: false
```

Comment:
Please, don't set `headers` and `isArray` properties because application using them by default. Just set the `method` like this example: `'send': { method: 'POST' }`